src/main/java/com/epam/prd/jjop/helpers/Object/Equals.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.prd.jjop.helpers.Object;
```

Comment:
An inner package name breaks java naming conversion (no upper case letters in packages)